src/documents/SVGAndAccessibility.md

```diff
@@ -0,0 +1,100 @@
+
+## What Possibilities Does SVG Offer for Inclusive Design?
+
+Because SVG is expressed in pure markup and then interpeted and rendered by the browser in a similar manner to HTML, it has some interesting potential for inclusive design:
+
+* It's a technology based on web standards, and requires no separate plug-ins to support
+* Because it forms part of the [Document Object Model](https://en.wikipedia.org/wiki/Document_Object_Model), it can be transformed by client-side code that interacts with the DOM, giving it potential for adaptation into alternative formats or reuse
+* SVG code is highly portable and can be shared or embedded in another context simply by embedding its markup
+* Individual or grouped polygons can have different markup, such as different ARIA attributes; this can help support building more complex interactions
+* The structure, content and presentation of SVG can be separated, similar to best practices around HTML
+
+## How Do We Support the Accessibility of SVG Content?
+
+Techniques and approaches to supporting accessible SVG include:
+
+* Use `aria-role` to describe the role of SVG content in the page. This can avoid unexpected or confusing behaviour by ATs interpreting the markup
+  * for example, a fuctionally static image created from a combination of SVG elements may have individual elements read by a screen reader. Assigning `aria-role="image"` can prevent this behaviour and allow a screen reader to interpret and convey the image as a single image element, with appropriate alternate text description
+* Use the `title` and `desc` elements made available by SVG for providing text alternatives
+* Make use of `aria-labelledby`, `aria-labels`, `aria-describedby`, `aria-describes` and other ARIA attributes to indicate related elements both within the SVG and between the SVG and the rest of the document
+* If using SVG for animated or interactive elements, use ARIA attributes and provide keyboard alternatives to interaction; some guidance for games and simulations using SVG can be found at  the[Inclusive Learning Design Handbook entry on Web Games and Simulations](/WebGamesAndSimulations.html)
+* Where possible, consider the use of [CSS to manage the visual appearance of SVG elements](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_started/SVG_and_CSS) as a means of separating presentation from structure and content; this will make adaptation to alternative presentations easier
+
+## Caveats
+
+SVG is a large and complex non-HTML technology, albeit one that is supported by modern web browsers well enough to offer many useful possibilities. However
+
+* Complex or advanced uses of SVG should be considered with [reference to browser support for specialized features](http://caniuse.com/#cats=SVG)
+* If features can be implemented or information presented using only HTML, this is probably preferable to using SVG; however, SVG has many capabilities unavailable in HTML for handling advanced graphics rendering
```

Comment:
If we don't have some good use cases, instead of removing the entire bullet point, probably only remove from "however" because I think the first half is certainly valid just by considering the benefit of using html and the complexity of SVG.